src/compose.js

```diff
@@ -10,14 +10,12 @@
  */
  
 export default function compose(...funcs) {
-  return (...args) => {
-    if (funcs.length === 0) {
-      return args[0]
-    }
-
-    const last = funcs[funcs.length - 1]
-    const rest = funcs.slice(0, -1)
-
-    return rest.reduceRight((composed, f) => f(composed), last(...args))
+  if(funcs.length){
+    const last = funcs[funcs.length - 1];
+    const rest = funcs.slice(0, -1);
+    
+    return (...args) => rest.reduceRight((composed, f) => f(composed), last(...args));
+  }else{
```

Comment:
Just started using github
why The Travis CI build Report an error always ?
i dont use local editor ,only use explore
ok, put commented